CMakeLists.txt

```diff
@@ -74,7 +74,7 @@ if(WIN32)
     add_subdirectory(src/qgittag)
 endif()
 
-find_package(Qt5 REQUIRED COMPONENTS Widgets LinguistTools Concurrent)
+find_package(Qt5 REQUIRED COMPONENTS Core Widgets LinguistTools Concurrent)
```

Comment:
Why `Core` is required? If you require `Widgets`, `Core` is automatically included.

Reply:
Because `Qt5::qmake` is defined in Qt5CoreConfigExtras.cmake

Reply:
I assume it should work becaue `Widgets` already imports `Core`. I tried it on my machine and it works.